packages/react-core/src/components/Alert/Alert.d.ts

```diff
@@ -0,0 +1,18 @@
+import { SFC, HTMLProps, ReactNode } from 'react';
+import { Omit, OneOf } from '../../typeUtils';
```

Comment:
Nice catch

Reply:
Linter is not running against d.ts files. I know that eslint is supposed to support that, but I have never tried it.  I can research it.  Otherwise it may not hurt to add tslint to the repo. Maybe even add a tslint preset for any ts projects as well similar to our eslint package.